lib/offline/download_manager.js

```diff
@@ -237,6 +237,11 @@ shaka.offline.DownloadManager.prototype.updateProgress_ = function() {
       (this.givenBytesTotal_ + this.bandwidthBytesTotal_);
 
   goog.asserts.assert(this.manifest_, 'Must not be destroyed');
+  /**
+   * TODO(hochhaus): Resolve circular dependency.
```

Comment:
We pushed a fix for #431, so you should be able to rebase your work and see this fixed.